pkg/storegateway/indexheader/reader_pool_test.go

```diff
@@ -17,6 +17,8 @@ import (
 	"github.com/prometheus/prometheus/model/labels"
 	"github.com/stretchr/testify/require"
 
+	"github.com/grafana/dskit/gate"
+
 	"github.com/thanos-io/objstore/providers/filesystem"
 
 	"github.com/grafana/mimir/pkg/storage/tsdb/block"
```

Comment:
These imports seem improperly formatted, but I'm not sure why the linter isn't picking it up. We usually group the imports into 3 groups - standard library, 3rd party packages, same-project packages. [Here is an example](https://github.com/grafana/mimir/blob/ae653d687cbc419db89759b6c81f4e9f49150ad2/pkg/storegateway/indexheader/stream_binary_reader.go#L8-L26)